index.html

```diff
@@ -172,6 +173,12 @@ <h2>Definitions</h2>
         When set, this [=preference=] indicates that the person expects to browse the Web with
         [=do-not-sell-or-share interactions=].
       </p>
+      <p>
+        <dfn data-lt="cross-party ad targeting|">Cross-party targeted
+        advertising</dfn> means showing a person advertisements, where the advertisement is selected
+        based on data about that person that was gathered from any party other than the
```

Comment:
```suggestion
        based on data about that person that was gathered over time and across parties.
```

Reply:
I also heard that we should try not to invent new language, so let's make this resemble the text of the various laws we're discussing (but use "across parties" instead of "across nonaffiliated websites", to be consistent with Ben's suggestions).

Reply:
"across parties" seems wrong.  Are we not talking "across [contexts](https://w3ctag.github.io/privacy-principles/#context)"